pkg/localConfig/localConfig.go

```diff
@@ -71,6 +85,21 @@ func (lc *LocalConfig) Set(key string, value string) error {
 	return nil
 }
 
+func InitLocalConfigFile() error {
+	configHome, configName, configType, err := setViperConfig()
```

Comment:
make sure to keep the comment with the references to the issue and pr in viper

Reply:
added.